template-contributing-guide.md

```diff
@@ -0,0 +1,341 @@
+
+
+### Template roles and resources
+
+As you work on contributing templates to our project, various resources and members of our community are available to help you along the process. These include:
+
+- **Templateers** - Any individual who contributes to the Good Docs Project (including you!).
+- **The templates coordinator** - This templateer oversees our overall template development process as a project manager and provides assistance to contributors working on templates.
+- **Template mentors** - This group of experienced templateers lead template writing working groups where you can receive guidance and mentorship while working on your template.
+- **Community reviewers** - This group of templateers are available to review templates during the workshop phase. They include members of your template workshop but also members from the larger Good Docs Project community who have volunteered to help review templates.
+- **Pull request reviewers** - This group of experienced templateers review and approve pull requests submitted to the templates repository.
+
+In addition to these roles, these resources are available to help you as you contribute templates to our project:
+
+- **Working groups** - Our project is organized into several working groups that meet on a regular basis to work on the project’s key initiatives. To effectively contribute to a project, consider joining one of these working groups.
+- **Templates issue list** - All the templates that are actively being worked on or which could be worked on have a corresponding issue in the templates repository. Use the issue list or the kanban board to find a template to work on and track your template’s progress as it moves throughout the template writing phases.
+
+These roles and resources (including the names of the individuals currently serving in these roles) are explained in the [Template Roles and Resources](template-roles-and-resources.md) guide.
+
+
+### Required deliverables
+
+Currently, for a template set to be considered complete, each template set should have these documentation deliverables:
+
+<table>
+  <tr>
+    <th>Deliverable</th>
+    <th>Description and purpose</th>
+    <th>Required?</th>
+  </tr>
+  <tr>
+    <td>Raw template</td>
+    <td>The raw template is the basic template for the type of document you are creating. It provides a rough outline of the suggested content and a few embedded writing tips for how to fill in the different sections of the template.</td>
+    <td>Yes, required</td>
+  </tr>
+  <tr>
+    <td>Template guide</td>
+    <td>This guide provides a deeper explanation of how to fill in the raw template. It provides a lightweight introduction to the purpose of this documentation and explains how to fill in each section of the document. Additional content may be included as needed.</td>

```

Comment:
If anything should be called "Before you start", shouldn't it be the guide? I actually think guide is a good name, but note from the description here that this is the essential info the user needs before they fill in the template.

Reply:
See my reply to your previous comment.

Reply:
I've updated our document based on what we agreed to in last week's US general meeting. Resolving.